doxyqml/doxyqml.py

```diff
@@ -107,12 +109,18 @@ def find_classname(qml_file):
     if modulename:
         classname = modulename + '.' + classname
 
+    if namespace:
+        classname = '.'.join(namespace) + '.' + classname
+
     return classname, classversion
 
 def main():
     args = parse_args()
 
     name = args.qml_file
+    namespace = []
+    if args.namespace:
+        namespace = args.namespace
```

Comment:
You can simplify this code to `namespace = args.namespace` by defining the default value of the "namespace" option to an empty list (`default=[]`) in `parse_args()`.

Reply:
Yep, that's much cleaner. Fixed.